guides/common/modules/proc_disabling-dhcp-for-integration.adoc

```diff
@@ -0,0 +1,28 @@
+[id="disabling-dhcp-for-integration"]
+= Disabling DHCP for integration
+
+If you want to manually manage a DHCP service, you must prevent {Project} from maintaining this service on the operating system and disable orchestration to avoid errors.
+
+[NOTE]
+====
+Disabling DHCP in {Project} does not remove the related backend service on the operating system.
+====
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets*.
+. For each subnet that is associated with the DHCP {SmartProxy}:
+.. Select the subnet.
+.. On the *{SmartProxies}* tab, clear the *DHCP {SmartProxy}* field.
+.. Click *Submit*.
+. On {ProjectServer} and {SmartProxyServer}, enter:
```

Comment:
```suggestion
. On {ProjectServer} and {SmartProxyServer}:
```
But to be consistent with the other procedures, you might want to go with "Perform these steps on [...]" the way you're describing it there.